samples/python/topology/simple/filter_echo.py

```diff
@@ -26,6 +27,7 @@ def main():
     """
     
     topo = Topology("filter_echo")
+#    topo.checkpoint_period(1, TimeUnit.MINUTES)
```

Comment:
Don;t change this existing examples .

Reply:
Sorry, unintended change.  I will change it back.